.github/workflows/pr.yml

```diff
@@ -0,0 +1,49 @@
+name: sync zoekt
+on:
+  push:
+    branches:
+      - main
+jobs: 
+  sync-zoekt:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+        with: 
+          repository: 'sourcegraph/sourcegraph' 
+          ref: 'main'
+      - uses: actions/setup-go@v2
+        name: 'Update zoekt'
+        with: { go-version: '1.17' } 
+      - run: go mod download
+      - run: ./dev/zoekt/update
+      - uses: peter-evans/create-pull-request@v3
+        name: 'Create PR'
+        id: pr
+        with: 
+          token: ${{ secrets.GH_TOKEN }}
+          title: Update Zoekt
+          commit-message: 'update to latest version of zoekt'
+          body: |
+            This PR is autogenerated to update zoekt.
+
+            ## Test Plan
+
+            Testing is done on the zoekt repo. All checks on this PR are sufficient.
+
+          branch: 'update-zoekt/zoekt'
+          delete-branch: 'true'
+          reviewers: 'sourcegraph/search-core'
+          base: 'main'
+          labels: 'automerge'
```

Comment:
```suggestion
```

Looks like we have enable "real" automerge below, so we shouldn't need a label on top